apps/prairielearn/src/lib/manualGrading.ts

```diff
@@ -363,6 +364,7 @@ async function recomputeInstanceQuestions(
         null, // check_modified_at,
         { manual_rubric_data: instance_question },
         authn_user_id,
+        instance_question.is_ai_graded, // is_ai_graded
```

Comment:
This comment doesn't do anything, it's entirely clear what is being passed in here from the name.

```suggestion
        instance_question.is_ai_graded,
```